tests/conftest.py

```diff
@@ -3,7 +3,7 @@
 
 Fixtures defined here are available to any test in Firecrown.
 """
-
+# pylint: disable=too-many-lines
```

Comment:
Pylint is saying we have 1021/1000 lines. I do not know if I should change this file to try to reuse some functions or we can ignore this warning